### DISCUSSION TITLE
some accounts is suspended

### DISCUSSION THREAD

**Discussion by ep-fangrongchang** [community/community#50562] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

We purchased a 60-seat GitHub Copilot and registered a GitHub account using our enterprise email. However, the registered account was partially suspended, which has prevented us from using GitHub Copilot properly. This has greatly affected our user experience. Is there any solution to this issue?

**Comment by Akash1134** [comment]
We're sorry for the trouble here. The best place to go for answers to account-specific questions like this one is by opening a ticket on our Support page here: https://support.github.com/contact  Please open a ticket with them and they should be able to help you get this sorted :)

**Reply by ep-fangrongchang** [reply]
Thank you! I submitted a ticket two days ago with ticket number 2065713, but have not received a response yet.

**Reply by Awesome-GJK** [reply]
I have a question, this Support page(https://support.github.com/contact) is it really useful, will anyone read it?

**Reply by tmpyc0304** [reply]
I also want to know if there anyone will read my ticket. I already waiting for a month. I can't do anything without my account.

**Comment by Awesome-GJK** [comment]
My friend had a similar experience, and his account was suspended for no reason without any email notification. He appealed too, but no one contacted him.

**Reply by ghost** [reply]
@Awesome-GJK did your friend get the account back? I'm on the same situation.
